samples/zigbee/light_switch/README.rst

```diff
@@ -56,6 +59,20 @@ Button 3:
 Button 4:
     When pressed, toggles the Finding & Binding mode on the local endpoint on or off.
 
+Sleepy behavior and Power Consumption
+*************************************
+
+Sleepy behavior of End Device allow to lower device power consumption and can be enabled by pressing **Button 3** while the light switch sample is booting.
+It allows the Zigbee stack to enter sleep state for a period of time for which nothing is scheduled and also power off some peripherals.
```

Comment:
I'd remove `and also power off some peripherals` part and add a sentence describing what does it mean that the stack enters sleep state.
Something like:
```
When the stack enters sleep state it releases all peripherals and suspends the Zigbee thread for an expected inactivity time.
The stack operation will be resumed after that time or if a Zigbee scheduler API is used by other thread.
```
I'm not sure if we need to say anything about the CPU idle here. It is up to the system to define what does the idle task do.

Reply:
Changed after tech writer review